tests/commands/run_test.py

```diff
@@ -766,6 +766,47 @@ def test_lots_of_files(store, tempdir_factory):
         )
 
 
+def test_no_textconv(cap_out, store, repo_with_passing_hook):
+    # git textconv filters can hide changes from hooks
+    with open('.gitattributes', 'w') as fp:
+        fp.write('*.jpeg diff=empty\n')
+
+    with open('.git/config', 'a') as fp:
+        fp.write('[diff "empty"]\n')
+        fp.write('textconv = "python -c print()"\n')
```

Comment:
this should use `sys.executable` (or maybe even bash) so it doesn't fail on systems with only `python3`

Reply:
Switched to `sys.executable`

Reply:
eheh I think it might have quoting issues now /o\  we'll see what CI says

Reply:
I would just use `echo` probably

Reply:
`echo` is only a shell builtin on my system; `printf` is binary, but it will require printing its args, which can cause a textconv difference since `git diff` uses a temporary filename for comparison

I added `shlex.quote`...

Reply:
I'm surprised you don't have /bin/echo, it's posix specced iirc

Reply:
`/bin/echo` exists yes, I was just being slow and only checked `/usr/bin/`.

I have changed the test to use `true`.